From 75c8f62795dbdfb387884f8b86548f1087daa246 Mon Sep 17 00:00:00 2001
From: Jung-Kwang Lee <ntoskrnl7@gmail.com>
Date: Mon, 18 May 2026 16:21:24 +0900
Subject: gin: add worker main wrappable tags

---
 gin/public/wrappable_pointer_tags.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gin/public/wrappable_pointer_tags.h b/gin/public/wrappable_pointer_tags.h
index 0f649fa562cef..66875adf7afa9 100644
--- a/gin/public/wrappable_pointer_tags.h
+++ b/gin/public/wrappable_pointer_tags.h
@@ -79,6 +79,7 @@ enum WrappablePointerTag : uint16_t {
   kIndigoContext,                // indigo::IndigoContext
   kElectronApp,                         // electron::api::App
   kElectronDataPipeHolder,              // electron::api::DataPipeHolder
+  kElectronDedicatedWorkerContext,      // electron::api::DedicatedWorkerContext
   kElectronDebugger,                    // electron::api::Debugger
   kElectronEvent,                       // gin_helper::internal::Event
   kElectronExtensions,                  // electron::api::Extensions
@@ -90,6 +91,7 @@ enum WrappablePointerTag : uint16_t {
   kElectronReplyChannel,                // gin_helper::internal::ReplyChannel
   kElectronScreen,                      // electron::api::Screen
   kElectronServiceWorkerContext,        // electron::api::ServiceWorkerContext
+  kElectronSharedWorkerContext,         // electron::api::SharedWorkerContext
   kElectronSession,                     // electron::api::Session
   kElectronTray,                        // electron::api::Tray
   kElectronWebRequest,                  // electron::api::WebRequest
